Source/Tools/LinuxEmulation/LinuxSyscalls/FileManagement.cpp

```diff
@@ -829,6 +844,13 @@ uint64_t FileManager::Openat2(int dirfs, const char* pathname, FEX::HLE::open_ho
 }
 
 uint64_t FileManager::Statx(int dirfd, const char* pathname, int flags, uint32_t mask, struct statx* statxbuf) {
+  const bool Follow = (flags & AT_SYMLINK_NOFOLLOW) == 0;
+  if (!Follow && IsSelf(pathname)) {
```

Comment:
Would it be better for IsSelf to also take an `int flags` so we deduplicate the `NOFOLLOW` check?

Reply:
Passed in the flags instead